docs/docusaurus.config.js

```diff
@@ -0,0 +1,190 @@
+          {
+            to: '/getting-started',
+            from: ['/tutorial/getting-started','/','/concepts', '/concepts/basic', '/concepts/advanced'],
+          },
+        ],
+      },
+    ],
+  ],
+
+  themeConfig:
+    /** @type {import('@docusaurus/preset-classic').ThemeConfig} */
+    ({
+      colorMode: {
+        respectPrefersColorScheme: true,
+      },
+      navbar: {
+        title: "Slack Developer Tools",
+        logo: {
+          alt: 'Slack logo',
+          src: 'img/slack-logo.svg',
+          href: 'https://slack.dev',
+          target : '_self'
+        },
+        items: [
+          {
+            type: 'dropdown',
+            label: 'Bolt',
+            position: 'left',
+            items: [
+              {
+                label: 'Java',
+                to: 'https://slack.dev/java-slack-sdk/guides/bolt-basics',
+                target: '_self',
+              },
+              {
+                label: 'JavaScript',
+                to: 'https://slack.dev/bolt-js',
+                target: '_self',
+              },
+              {
+                label: 'Python',
+                to: 'https://slack.dev/bolt-python',
+                target: '_self',
+              },
+            ]
+          },
+          {
+            type: 'dropdown',
+            label: 'SDKs',
+            position: 'left',
+            items: [
+              {
+                label: 'Java Slack SDK',
+                to: 'https://slack.dev/java-slack-sdk/',
+                target: '_self',
+              },
+              {
+                label: 'Node Slack SDK',
+                to: 'https://slack.dev/node-slack-sdk/',
+                target: '_self',
+              },
+              {
+                label: 'Python Slack SDK',
+                to: 'https://slack.dev/python-slack-sdk/',
+                target: '_self',
+              },
+              {
+                label: 'Deno Slack SDK',
+                to: 'https://api.slack.com/automation/quickstart',
+                target: '_self',
+              },
+            ]
+          },
+          {
+            type: 'dropdown',
+            label: 'Community',
+            position: 'left',
+            items: [
+              {
+                label: 'Community tools',
+                to: 'https://slack.dev/community-tools',
+                target: '_self',
+              },
+              {
+                label: 'Slack Community',
+                to: 'https://slackcommunity.com/',
+                target: '_self',
+              },
+            ]
+          },
+          {
+            to: 'https://api.slack.com/docs',
+            label: 'API Docs',
+            target: '_self',
+          },
+          {
+            type: 'localeDropdown',
+            position: 'right',
+          },
+          {
+            'aria-label': 'GitHub Repository',
+            'className': 'navbar-github-link',
+            'href': 'https://github.com/slackapi',
```

Comment:
```suggestion
            'href': 'https://github.com/slackapi/bolt-python',
```

I think the kind readers will be most interested in the repo of the current docs- might be a nice change to have?

Reply:
good point good point i was prob being lazy and keeping it the same for copy/paste